test/about/historyTest.js

```diff
@@ -132,14 +136,12 @@ describe('about:history', function () {
     })
     it('selects multiple contiguous rows when shift clicked', function * () {
       yield this.app.client
-        .tabByUrl(aboutHistoryUrl)
-        .loadUrl(aboutHistoryUrl)
         .click('table.sortableTable td.title[data-sort="Brave"]')
         .keys(Brave.keys.SHIFT)
         .click('table.sortableTable td.title[data-sort="https://www.youtube.com"]')
         .waitForVisible('table.sortableTable tr.selected td.title[data-sort="Brave"]')
         .waitForVisible('table.sortableTable tr.selected td.title[data-sort="https://www.youtube.com"]')
-        .waitForVisible('table.sortableTable tr.selected td.title[data-sort="https://brave.com/test"]')
+        .waitForVisible('table.sortableTable td.title[data-sort="https://brave.com/test"]')
```

Comment:
Can you please explain why `selected` should be removed? I just run `npm run test -- --grep="about:history"` on a local machine and it's working correctly. Also master is not failing. Same goes for all `selected` removed bellow.
Thank you